pulp_container/app/models.py

```diff
@@ -565,6 +642,25 @@ def remove_pending_content(self, repository_version):
         self.pending_manifests.remove(*Manifest.objects.filter(pk__in=added_content))
 
 
+class ContainerPullThroughDistribution(Distribution, AutoAddObjPermsMixin):
+    """
+    TODO: Add permissions.
+    """
```

Comment:
May I use the same permissions that are currently used by `ContainerDistribution`? The result would look like this:
```
        permissions = [
            ("pull_containerdistribution", "Can pull from a registry repo"),
            (
                "manage_roles_containerdistribution",
                "Can manage role assignments on container distribution",
            ),
        ]
```

Reply:
Same stands for `ContainerRemote`:
```
        permissions = [
            (
                "manage_roles_containerremote",
                "Can manage role assignments on container remote",
            ),
        ]
```

Reply:
And, of course, the same for ViewSets.